.idea/compiler.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Respektive jsi nechtěl commitovat celou .idea složku - ne všichni používají ideu, takže v tom repozitáři to je zbytečný. Metody záchrany: git rm a kámoši